extensions/azure/cosmos-common/src/main/java/org/eclipse/dataspaceconnector/cosmos/azure/CosmosDbApi.java

```diff
@@ -0,0 +1,17 @@
+package org.eclipse.dataspaceconnector.cosmos.azure;
+
+import com.azure.cosmos.models.SqlQuerySpec;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public interface CosmosDbApi {
+
+    void createItem(Object item);
+
+    @Nullable Object queryItemById(String id);
+
+    List<Object> queryAllItems();
+
+    List<Object> queryItems(SqlQuerySpec query);
```

Comment:
Let's try and keep Cosmos-SDK classes out of this interface. Why not simply pass in a String?